src/libraries/System.Security.Cryptography.X509Certificates/tests/PublicKeyTests.cs

```diff
@@ -355,7 +370,7 @@ public static void TestKey_RSA384_ValidatesSignature()
             }
         }
 
-        [Theory, MemberData(nameof(BrainpoolCurves))]
+        [ConditionalTheory(nameof(SupportsBrainpool)), MemberData(nameof(BrainpoolCurves))]
```

Comment:
Hmm.  macOS doesn't support Brainpool, nor does Windows7.  I think this suggests we're throwing a PNSE out of Android where we throw a CryptographicException elsewhere.

Reply:
Oh I added this condition preemptively when I added the other condition on SupportsBrainpool. I'll remove this one since it doesn't make a difference in the test.

Reply:
Well, the other instance (in this class) would have the same "where's the difference between Android and Windows 7?"